src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -889,23 +889,17 @@ so that we can apply CSS-filters to change the arrow color in themes */
 }
 
 .search-results > a {
-	display: block;
+	display: flex;
 	/* A little margin ensures the browser's outlining of focused links has room to display. */
 	margin-left: 2px;
 	margin-right: 2px;
 	border-bottom: 1px solid var(--border-color);
+	gap: 1em;
 }
 
 .search-results > a > div {
-	display: flex;
-	flex-flow: row wrap;
-}
-
-.search-results .result-name, .search-results div.desc {
-	width: 50%;
-}
-.search-results .result-name {
-	padding-right: 1em;
+	flex: 1;
+	overflow: hidden;
```

Comment:
Why do you need this? If something is too long, its content should just go on the next ine as it currently does. Is it still the case?

Reply:
Then shouldn't it be:

```css
text-overflow: ellipsis;
```

?

Reply:
Thanks!